README.md

```diff
@@ -1,374 +1,15 @@
-<p align="center">
-<img width="300" height="280"
-src="screenshots/Trapheus.png">
-</p>
-<p align="center">
-<b>Restore RDS instances in AWS without worrying about client downtime or configuration retention.</b><br/>
-<sub>Trapheus can restore individual RDS instance or a RDS cluster.
-Modelled as a state machine, with the help of AWS step functions, Trapheus restores the RDS instance in a much faster way than the usual SQL dump preserving the same instance endpoint and confgurations as before.
-</sub>
-</p>
-<p align="center"><a href="https://circleci.com/gh/intuit/Trapheus"><img src="https://circleci.com/gh/intuit/Trapheus.svg?style=svg" alt="TravisCI Build Status"/></a>
-<a href = "https://coveralls.io/github/intuit/Trapheus?branch=master"><img src= "https://coveralls.io/repos/github/intuit/Trapheus/badge.svg?branch=master" alt = "Coverage"/></a>
-  <a href="https://www.serverless.com"><img src="https://github.com/intuit/Trapheus/assets/112308320/c7764b65-fb72-46ed-824b-c82c6374ec8b" alt="serverless badge"/></a>
-  <a href="https://github.com/intuit/Trapheus/releases"><img src="https://img.shields.io/github/v/release/intuit/trapheus.svg" alt="release badge"/></a>
-</p>
+<div align="center">
 
- <img src="https://ch-resources.oss-cn-shanghai.aliyuncs.com/images/lang-icons/icon128px.png" width="22px" /> [English](README.md) |  [简体中文](./docs/README.zh-CN.md) | [français](./docs/README.fr.md)
+# Trapheus & TrapheusAI
 
+**Trapheus** is your reliable solution for automated database snapshot and restoration, built to handle complexities with ease within the AWS ecosystem. It stands out with its robust state machine approach, designed for both instance and cluster databases, ensuring your data is not just secure but also retrievable and consistent with minimal downtime.
 
-* **Important:** this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS  pricing page](https://aws.amazon.com/pricing/) for details.
+👉 [View Trapheus Documentation](\docs\README.md)
```

Comment:
Instead of ```View Trapheus Documentation``` , can it be ```Trapheus Docs``` same for TrapheusAI

Reply:
> Instead of `View Trapheus Documentation` , can it be `Trapheus Docs` same for TrapheusAI

Okay, will modify this. Also would like to go ahead and add the 'Prepare your environment' section present in Trapheus readme in the main readme? As I feel individuals should not really need to visit either documentation to get started with Trapheus code base?

Reply:
Both the repositories will have different way fo installation. I feel we can provide some content in terms of "what" Trapheus solves for in the world of data and then we can link them to their respective docs. Let me know if you think it would be meaningful?

Reply:
Have made the minor fix. Feel free to review this. I think some context of what each of them does has already been provided. If you'd like me to add something specific please let me know.

Reply:
Thanks @yash-1o1  I will review this today and possibly modify this a bit